packages/cli/src/models/status/StatusFetcher.ts

```diff
@@ -1,117 +1,120 @@
-import { ZWeb3, Contracts, Logger, FileSystem as fs, bytecodeDigest, bodyCode, constructorCode } from 'zos-lib'
+import { ZWeb3, Contracts, Logger, FileSystem as fs, bytecodeDigest, bodyCode, constructorCode } from 'zos-lib';
```

Comment:
I don't see this file completely ported. Maybe you forgot to push a commit? If you are planning to finish this port in another PR, please uncheck it in https://github.com/zeppelinos/zos/issues/434 until you tackle it again so we don't forget to to port it in the future :slightly_smiling_face:

Reply:
Yep, I'm currently working on it, as well as on all the files marked on the plan. Don't worry, if this is merged, I take notes of what I'm doing, so I'd know exactly where and how to continue with the marked files.